README.md

```diff
@@ -34,7 +34,7 @@ kubectl apply -f ./workloadscluster/certmanageringress.yaml
 ```
 
 - retrieve the tailnet IP address of the cert-manager metrics endpoint (you
-might need to wait a bit for Tailscale operator to set the it up):
+  might need to wait a bit for Tailscale operator to set the it up):
```

Comment:
sorry, my stupid auto-formatting in vscode was going wild 😅

Reply:
👍🏼 please do correct wording, formatting etc if you see things that could be better, I imagine if folks ask how it's set up we might want to show them the yamls in the repo.